README.md

```diff
@@ -97,6 +97,45 @@ You can now visit your wagtail site at the URL provided by `fly`. We strongly re
 
 The database and user-uploaded media are stored in the attached volume. To save costs and improve efficiency, the app will automatically stop when not in use, but will automatically restart when the browser loads.
 
+#### Divio Cloud
+
+[![Deploy to Divio](https://docs.divio.com/deploy-to-divio.svg)](https://control.divio.com/app/new/?template_url=https://github.com/divio/getting-started-with-payload/archive/refs/heads/main.zip)
+
+Easily deploy your application to [Divio Cloud](https://www.divio.com/) using the steps below:
+
+1. **Getting Started**  
+   Follow the [Getting Started](#getting-started) instructions to set up your project locally.
+
+2. **Push Your Repository**  
+   Upload your project to GitHub or another Git provider.
+
+3. **Create a New Application**  
+   Log in to the [Divio Control Panel](https://control.divio.com/) and create a new application and
+
+   - Choose "**I already have a repository**.".
+   - Connect your Git provider and proceed by clicking "**Next**.".
+   - Give your application a suitable name and select the "**Free Trial**" plan, then click **"Create application."**.
+   
+   Your application will be created with two environments: **Test** and **Live**.
+
+4. **Add a Database service**
+   From the **Services** view of your application, add a [database](https://docs.divio.com/introduction/aldryn-django/django-05-database/) service.
+
+5. **Deploy Your Application**  
+   From the "Environments" view, click "**Deploy**" on the **Test** environment. Once the deployment completes, access your site using the "Env URL" link.
+
+6. **Additional Configuration**  
+   **Migrations and Environment Variables**:
+   
+   To automatically run migrations on every deployment, add a "Release command" within the **Settings** section of your application with the value `python manage.py migrate`. 
+You can add additional commands as needed.
+
+   Use the **Env Variables** section to set variables such as `SECRET_KEY` for the test and live environments.
+
+   **Media Storage**: From the **Services** view of your application, add an [object storage](https://docs.divio.com/reference/work-media-storage/) to store user-uploaded files.
+
+   For further application-specific configurations, update your [`settings.py`](https://github.com/divio/getting-started-with-wagtail/blob/main/mysite/settings/base.py) accordingly.
```

Comment:
From my reading, the current application isn't actually compatible with Divio's media storage, since it uses differnet environment variables - could support be added to this template for them, rather than pointing people to modifying the template.

If the user needs to modify the template to make it work with the platform, it's not really a "supported" platform, it's just mentioned in the docs.

Reply:
Thank you for clarifying this! I’ve reviewed the discussion, and I noticed you mentioned:

"Are the `DEFAULT_*` environment variables automatically set by Divio? If so, I'm not opposed to adding native support for them to the template, given it's a graceful fallback."

I can confirm that yes, the `DEFAULT_*` environment variables are set by Divio. If adding native support for them to the template is an option, that would be greatly appreciated. Would this be a workable solution?

Reply:
Yep adding native support for them sounds useful to me, so long as it doesn't interfere with the existing allowed variables.

Reply:
Glad to hear that! How would you like to proceed? Would you like us to propose the necessary updates to the template, or would you prefer to handle them on your end?